### PR TITLE
[Gpu Channel] Enable gpu channel for rasterization.

### PR DESCRIPTION
**Pull request by Sunwoo-Nam**
Enable gpu compositing. And synchronize shared memories for
gpu command buffer and rasterized data between browser and renderer process.

### CODE REVIEW

File: gpu/command_buffer/client/cmd_buffer_helper.cc [hunk 1]
```diff
@@ -197,6 +213,17 @@ void CommandBufferHelper::OrderingBarrier() {
...
+    // If ring_buffer_ overflowed.
+    if (put_ < last_ordering_barrier_put_) {
+      size_t tail_size = ring_buffer_size_ - (last_ordering_barrier_put_ * kCommandBufferEntrySize);
+      mojo::SyncSharedMemoryHandle(ring_buffer_->backing()->GetGUID(), last_ordering_barrier_put_ * kCommandBufferEntrySize, tail_size);
+      mojo::SyncSharedMemoryHandle(ring_buffer_->backing()->GetGUID(), 0, put_ * kCommandBufferEntrySize);
+    } else {
+      size_t size = (put_ - last_ordering_barrier_put_) * kCommandBufferEntrySize;
+      mojo::SyncSharedMemoryHandle(ring_buffer_->backing()->GetGUID(), last_ordering_barrier_put_ * kCommandBufferEntrySize, size);
+    }
+#endif
```
Comment (hyowon) [inline gpu/command_buffer/client/cmd_buffer_helper.cc:234]: All the same with the above code. Add a new method?

Comment (hyowon) [reply gpu/command_buffer/client/cmd_buffer_helper.cc:234]: Only sync in renderer process.
if (base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("type") == "renderer")

**Review by hyowon — CHANGES_REQUESTED** [review]

File: gpu/command_buffer/client/cmd_buffer_helper.cc (hunk 1 above)
Comment (Sunwoo-Nam) [reply gpu/command_buffer/client/cmd_buffer_helper.cc:234]: I updated a commit.

**Review by hyowon — APPROVED** [review]
Now, both SW and GPU compositing run on Castanets mode. Nice PR!

**Review by is46-kim — APPROVED** [review]

**Pull request merged**